packages/material-ui/src/InputLabel/InputLabel.js

```diff
@@ -65,6 +65,9 @@ export const styles = theme => ({
         transform: 'translate(12px, 7px) scale(0.75)',
       },
     },
+    '&[hidden]': {
```

Comment:
Why is hidden not enough to hide the element that we need to force display none?

Reply:
because we use `display: block` in the root class.